src/Contact/View/page.twig

```diff
@@ -1,5 +1,30 @@
 {% include 'Common/View/header.twig' %}
-<section class="container">
-        <h1>Kontakt</h1>
+<section class="container contact-section">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css">
+        <form method="post" action="" class="form">   
+                <div>
+                <h1 class="form-title">NAPISZ DO NAS</h1> <br>
+                
+                        <div>
+                                <input class="form-input" type="text" name="name" placeholder="Imię i nazwisko" required>
+                        </div>
+                        <div>
+                                <input class="form-input" type="email" name="email" placeholder="E-mail" required>
+                        </div>
+                        <div>
+                                <input class="form-input"  type="text" name="theme" placeholder="Temat" required>
+                        </div>
+                        <div>
+                                <textarea class="form-input form-input--text" name="textarea" placeholder="Treść" required></textarea>
+                        </div>
+                        <div>
+                               
+                               <i  class="fas fa-check-circle" >  <input class="u-center-text submit-button" type="submit"  value="WYŚLIJ"  /> </i>
```

Comment:
I need to confess, that I didn't prepare for you css rules for adding check-circle icon. I decided, that it is useless. It great that you use external icon 👍 I you want come closer to appearance of mockup desing I will send you this original icon. But your solution is good too. 
Now how to place this icon inside box with input element? That is tricky. Would you like to try to make it?